src/main/scala/com/datastax/sparkstress/ReadTask.scala

```diff
@@ -5,24 +5,27 @@ import java.util.UUID
 import com.datastax.sparkstress.RowTypes.PerfRowClass
 import org.apache.spark.SparkContext
 import com.datastax.spark.connector._
+import org.joda.time.DateTime
 
 object ReadTask {
   val ValidTasks = Set(
-    "countall",
-    "aggregatecolor",
-    "aggregatecolorsize",
-    "filtercolorstringmatchcount",
-    "filterequalityqtycolorsizecount",
-    "filterequalityqtycolorsizecountfivecols",
-    "filterlessthanqtyequalitycolorsizecount",
-    "filterlessthanqtyequalitycolorsizefivecols",
-    "filterqtymatchcount",
+    "ftsallcolumns",
+    "ftsfivecolumns",
+    "ftsonecolumn",
+    "ftspdclusteringallcolumns",
+    "ftspdclusteringfivecolumns",
+    "jwcallcolumns",
+    "jwcpdclusteringallcolumns",
+    "jwcrpallcolumns",
+    "pdcount",
     "retrievesinglepartition"
   )
 }
 
 abstract class ReadTask(config: Config, sc: SparkContext) extends StressTask {
 
+  val uuidPivot = UUID.fromString("9b657ca1-bfb1-49c0-85f5-04b127adc6f3")
```

Comment:
Nah i forgot which clustering column came first, so I first had everything on order_number not order_time. I just left this hear so we can add double cluster column pushdowns if we feel like it.